chemie/shop/templates/shop/shop.html

```diff
@@ -0,0 +1,266 @@
+
+    </label>
+  </p>
+  {% endif %}
+  {% endif %}
+  {% endfor %}
+</div>
+{% if is_happy %}
+<div id="happy-hour-box">
+  <h4 style="text-align:center">Nå er det happy hour! 2 for 1 på alle varer med fyrverkeri</h4>
+</div>
+
+{% endif %}
+
+<!-- Display items as cards -->
+<div class="d-flex flex-wrap justify-content-evenly" id="card-items" style="justify-content: space-between">
+  {% for item in items %}
+  {% if is_happy and item.happy_hour_duplicate %}
+  <div class="p-2 item-box" id="{{item.category}}-{{item.id}}"
+    style="background-image: url(/static/images/fireworks_background.png);">
+    {% else %}
+    <div class="p-2 item-box" id="{{item.category}}-{{item.id}}">
+      {% endif %}
+      <div class="d-flex flex-column justify-content-between" style="height:100%">
+        <div class="p-2" style="max-height:50%; text-align:center">
+          {% thumbnail item.image "630x400" as pic %}
+          <img class="activator" style="max-width:10rem; max-height:9rem; margin-bottom:1rem" src="{{ pic.url }}">
+          {% endthumbnail %}
+        </div>
+
+        <div class="d-flex justify-content-around">
+          <div style="max-width: 7rem;">
+            {{ item.name }}
+          </div>
+          <div class="align-self-center">
+            {{ item.price }}
+            <img type="image/png" style="width:1.8rem" src="{% static "images/HC_coin.png" %}">
+          </div>
+        </div>
+
+        <div class="p-2">
+          <script type="text/javascript">
+            item_dict["{{item.name}}"] = 0; // appends item to dictionary
+          </script>
+          <div class="d-flex justify-content-between">
+            <div class="p-2">
+              <a class="2-p btn btn-danger" id="minus" onclick="minus('{{ item.name }}')"><i class="material-icons">
+                  remove
+                </i></a>
+              <input style="margin:0.1rem; width: 1.4rem; text-align: center" type="text" value="0" max="99"
+                id="item_{{item.name}}">
+              <a class="btn btn-success" id="plus" onclick="plus('{{ item.name }}')"><i class="material-icons">
+                  add
+                </i></a>
+            </div>
+            <div class="p-2">
+              <form action="#" method="POST">
+                {% csrf_token %}
+                <button id="{{ item.id }}" class="btn btn-secondary" style="background-color:white" type="submit"
+                  name="buy" value="{{ item.id }}-0" style="float:right"
+                  onclick="changeQuantity(this,'{{item.name}}');"><i class="material-icons"
+                    style="color:black;">add_shopping_cart</i></button>
+              </form>
+            </div>
+
+
+          </div>
+        </div>
+      </div>
+    </div>
+
+    {% endfor %}
+  </div>
+
+  <!-- Modal Structure -->
+
+  <!-- Modal -->
+  <div class="modal fade" id="cart-modal" tabindex="-1" role="dialog" aria-labelledby="exampleModalLabel"
```

Comment:
Burde denne modalen hatt et kryss i hjørnet for å lukke den? (Man kan bli forvirret og slette varen i stedet for å lukke modalen, når det kun er én vare)
<img width="514" alt="Screenshot 2019-05-05 at 12 36 45" src="https://user-images.githubusercontent.com/25224762/57192606-a17ecb00-6f32-11e9-8467-e300100695cf.png">

Kunne også vurdert å endre labelen til modalen til noe annet enn "exampleModalLabel" 😅

Reply:
endra iconet til en søppelkasse og la til en close modal button helt øverst til venstre
![image](https://user-images.githubusercontent.com/33221174/57194114-1b1eb500-6f43-11e9-99e3-348836f10d74.png)